### PR TITLE
[ocamformat] Update to 0.13.0

### PR DESCRIPTION
**Pull request by ejgallego**
We include the `version=0.13.0` field that should help users not to
use the wrong version. `disable=true` still seems a noop with `dune`.

There are some minor changes in the way comments are formatted; I'm
unsure if this is due to the `wrap-comments` option; as always; tweaks
to the config are most welcome.

### CODE REVIEW

**Review by Zimmi48 — COMMENTED** [review]
OK to me but #11732 needs to go in first because the current `nix-shell` gives ocamlformat 0.12.

**Comment by ejgallego** [comment]
> OK to me but #11732 needs to go in first because the current `nix-shell` gives ocamlformat 0.12.

ocamlformat 0.13 doesn't need a newer ocaml tho, so if you want to push the nix update here you are welcome

**Comment by Zimmi48** [comment]
@ejgallego The PR included both an update of nixpkgs and a bump of the OCaml version. The update of nixpkgs is what gives us ocamlformat 0.13.0. I've merged the PR now.

**Comment by ejgallego** [comment]
> @ejgallego The PR included both an update of nixpkgs and a bump of the OCaml version. The update of nixpkgs is what gives us ocamlformat 0.13.0. I've merged the PR now.

Thanks, let me rebase on top of that one.

**Comment by Zimmi48** [comment]
> There are some minor changes in the way comments are formatted; I'm
> unsure if this is due to the `wrap-comments` option; as always; tweaks
> to the config are most welcome.

Seems like this is no longer the case and thus we can remove the review request for micromega maintainers. You should also update your commit message.

There is also a conflict now. Can you rebase? I'll merge soon after.

**Comment by ejgallego** [comment]
Made this in top of #11860

**Comment by Zimmi48** [comment]
The dependency has been merged but there is a conflict now.

**Comment by ejgallego** [comment]
> The dependency has been merged but there is a conflict now.

Fixed, thanks.

**Review by Zimmi48 — APPROVED** [review]

**Pull request merged**
